homeassistant/components/binary_sensor/egardia.py

```diff
@@ -0,0 +1,83 @@
+"""
+Interfaces with Egardia/Woonveilig alarm control panel.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.egardia/
+"""
+import logging
+import asyncio
```

Comment:
Please sort 🔤 within groups standard, 3rd party and homeassistant.

Reply:
I hope I did it right ;)